path_planning/src/path_planning.cpp

```diff
@@ -85,8 +89,12 @@ class planner {
 			std::cout << "Goal point set to: " << x << " " << y << " " << z << std::endl;
 
 			if(set_start)
-				plan();
+				return plan();
+		}
+		else {
+			std::cout << "Path planner goal same as previous goal, not planning" << std::endl;
```

Comment:
Don't we send the same goal more than once with the service, if it fails?

Reply:
True, got rid of this conditional.